django/utils/datastructures.py

```diff
@@ -131,18 +131,21 @@ def get(self, key, default=None):
             return default
         return val
 
-    def getlist(self, key, default=None):
-        """
-        Returns the list of values for the passed key. If key doesn't exist,
-        then a default value is returned.
-        """
+    def _getlist(self, key, default=None):
```

Comment:
I would  be surprised to see that behavior for any method called `getlist` but maybe it's not unusual.